### PR TITLE
New Port Jump Off The Bridge

### PR DESCRIPTION
**Pull request by Fraxinus88**
# New Port for New Port Jump Off The Bridge

## Game Information
- **Title**: New Port Jump Off The Bridge
- **URL**: https://store.steampowered.com/app/1057780/Jump_Off_The_Bridge/

## Submission Requirements

CFW Tests:
[X] AmberELEC
[X] ArkOS
[X] JELOS/ROCKNIX
[X] MuOS

Architecture
[X] armHF
[X] Panfrost (Mainline)

Resolution Tests
Test all major resolutions:
[ ] 480x320
[X] 640x480
[X] 720x720 (RGB30)
[X] Higher resolutions (e.g., 1280x720) 

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

**Review by kloptops — APPROVED** [review]

File: ports/jumpoffthebridge/Jump Off The Bridge.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
```
Comment (Cebion) [inline ports/jumpoffthebridge/Jump Off The Bridge.sh:2]: Wrong header

```
XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}

if [ -d "/opt/system/Tools/PortMaster/" ]; then
  controlfolder="/opt/system/Tools/PortMaster"
elif [ -d "/opt/tools/PortMaster/" ]; then
  controlfolder="/opt/tools/PortMaster"
elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
  controlfolder="$XDG_DATA_HOME/PortMaster"
else
  controlfolder="/roms/ports/PortMaster"
fi```

File: ports/jumpoffthebridge/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+**Source:** https://store.steampowered.com/app/1057780/Jump_Off_The_Bridge/
+
+**Controls:**
+In-game tutorial
+Controls are rebindable with Start inside the Options Menu
+
+## Controls
+
+| Button | Action |
```
Comment (Cebion) [inline ports/jumpoffthebridge/README.md:13]: You have two Controls entries

File: ports/jumpoffthebridge/port.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Jump Off The Bridge",
+    "porter": [
+      "Fraxinus88"
+    ],
+    "desc": "Jump Off The Bridge is a platformer with an emphasis on difficulty and unique level design. Each level, the player is required to jump off the bridge, but dying in any other sense, is failure. You may encounter characters that require items, methodical puzzle levels, intensely difficult platforming, tough levels that test your reaction time, and even bosses. Jump Off The Bridge is a meticulously crafted world that avoids the overuse of previously introduced sprites and characters. Every level feels unique, and is memorable.",
+    "desc_md": null,
+    "inst": "Purchase the game from steam: https://store.steampowered.com/app/1057780/Jump_Off_The_Bridge/\nDownload the game and from your Steam library,  right click on Jump Off The Bridge >  Manage > Browse local files. Copy the data.win, audiogroup2.dat and audiogroup3.dat from the assets folder to /ports/jumpoffthebridge/gamedata",
```
Comment (Cebion) [inline ports/jumpoffthebridge/port.json:16]: In the future try to use ther inst_md for urls and such.
These will be shown on the website with proper links.

**Review by Cebion — CHANGES_REQUESTED** [review]
See comments

File: ports/jumpoffthebridge/Jump Off The Bridge.sh (hunk 1 above)
Comment (Fraxinus88) [reply ports/jumpoffthebridge/Jump Off The Bridge.sh:2]: whoopsies

File: ports/jumpoffthebridge/port.json (hunk 3 above)
Comment (Fraxinus88) [reply ports/jumpoffthebridge/port.json:16]: okies will do so with the next batch of PRs :p

**Pull request merged**